pages/partials/helpLinkPanel.ejs

```diff
@@ -0,0 +1,11 @@
+<div id="linkNav">
+  <% var url = assessment.help_link %>
+  <% if (!assessment.help_link.includes("https://")){%>
+    <% url = "https://" + assessment.help_link%>
+  <% } %>
```

Comment:
I don't think we should do this. Although it would be ill-advised, we don't necessarily want to prevent people from using `http://` links. If someone wants a protocol, they should include it in their URL.